lib/iris/cube.py

```diff
@@ -3368,8 +3368,7 @@ def aggregated_by(self, coords, aggregator, **kwargs):
 
         A "group coordinate" is a coordinate where repeating values represent a
         single group, such as a month coordinate on a daily time slice.
-        TODO: It is not clear if repeating values must be consecutive to form a
-        group.
+        Repeated values can form a group even if they are not consecutive.
```

Comment:
The word "can" suggests it doesn't always happen, perhaps "Repeated values will form a single group even ..."?